public/talos/v1.12/configure-your-talos-cluster/storage-and-disk-management/disk-encryption.mdx

```diff
@@ -38,7 +44,7 @@ Talos Linux supports four encryption methods, which can be combined together for
 
 > Note: `nodeID` encryption is not designed to protect against attacks where physical access to the machine, including the drive, is available.
 > It uses the hardware characteristics of the machine in order to decrypt the data, so drives that have been removed, or recycled from a cloud environment or attached to a different virtual machine, will maintain their protection and encryption.
-> 
+>
```

Comment:
while you're on it, we should add a note about TPM disk encryption and list of TPM registers to lock to

@frezbo can help here